README.md

```diff
@@ -1 +1,53 @@
-##Network Service
+# Network Service 
+
+The repo contains a Swift Package with the implementation of the HTTP network service for the iOS platform.
+
+⚠️ The Network service is under active development.
+
+## Table of contents
+
+* [General info](#general-info)
+* [Installation](#installation)
+* [Authors](#authors)
+
+## Current dev environment 
+
+| Dev environment| Ref link |
```

Comment:
Do we need the whole table here? Can we put just badges at the top of the README file?

Reply:
strictly speaking no, only the badges moved up